.github/workflows/node-ci.yml

```diff
@@ -153,13 +158,20 @@ jobs:
             - name: Checking out latest commit
               uses: actions/checkout@v4
 
+            # MUST be before we install node_modules as that depends on finding
+            # Cypress binaries in this cache!
+            - uses: actions/cache@v4
+              with:
+                  path: ~/.cache/Cypress
+                  key: cypress-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
+
```

Comment:
Cypress runs a post-install script (which we've allowed in `package.json`), but it doesn't seem to include downloading the Cypress binaries if we've cached `node_modules` (next step). By caching this [directory](https://docs.cypress.io/app/references/advanced-installation#Binary-cache), the post-install script is able to find a cached binary and connect it. 

On a clean install (or Cypress upgrade) the `node_modules` cache will be busted and the cypress npm package properly downloads the correct binary and caches it in `~/.cache/Cypress`.

Reply:
That's nice, I imagine that would speed things up a bit.

Reply:
On a random sampling of one other PR, [this workflow took](https://github.com/Khan/perseus/actions/runs/13247696841/job/36978127948) 1m 34s. 

On this PR, [this same workflow](https://github.com/Khan/perseus/actions/runs/13210137302/job/36881957372?pr=2202) took 1m 10s.  

So, yes, 24s faster!